server/bin/pbench-unpack-tarballs.sh

```diff
@@ -354,7 +355,7 @@ function do_work() {
         let end_time=$(timestamp-seconds-since-epoch)
         let duration=end_time-start_time
         # log the success
-        echo "${TS}: ${hostname}/${resultname}: success - elapsed time (secs): ${duration} - size (bytes): ${size}"
+        log_info "${TS}: ${hostname}/${resultname}: success - elapsed time (secs): ${duration} - size (bytes): ${size}"
```

Comment:
No, but I think it should.

Reply:
Can we debate that in another PR that makes those changes?  Here, it would be great if her changes can be made without any changes to the unit test output.

Reply:
OK - we can l;eave it for another PR.